src/tactic/find_unused.lean

```diff
@@ -0,0 +1,102 @@
+/-
+Copyright (c) 2020 Simon Hudon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Simon Hudon
+-/
+import tactic.core
+
+/-!
+This is a development file. When writing a new theory one often tries
```

Comment:
```suggestion
# find_unused

`find_unused` is a command used for theory development. 
When writing a new theory one often tries
```

Reply:
Does the module doc need to be updated to account for checking multiple files at once?